kornia/testing/__init__.py

```diff
@@ -306,19 +306,22 @@ def KORNIA_CHECK_SHAPE(x, shape: List[str]) -> None:
     KORNIA_CHECK_IS_TENSOR(x)
 
     if '*' == shape[0]:
-        start_idx: int = 1
+        shape_to_check = shape[1:]
```

Comment:
This is from #2070 -- maybe wait to merge that before this patch

Reply:
agree